test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import logging
+import sys
+
+import pytest
+
+sys.path.insert(0, sys.path[0] + "/test/cql-pytest")
+import nodetool
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, TextType, Column
+from test.pylib.util import unique_name
+
+logger = logging.getLogger(__name__)
+CONFIG = {"endpoint_snitch": "GossipingPropertyFileSnitch"}
+cluster_config = [
+    ([1, 2], 1),
+    ([1, 1, 2, 2], 2)
+]
```

Comment:
Please move cluster_config to right above the place it is used, it will be clearer.